### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Add simple request / response / error logging to be able to monitor requests.

## Changes
- Added simple logging.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/logger.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const logger = winston.createLogger({
+    level: 'info',
+    format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.json()
+    ),
+    transports: [
+        new winston.transports.File({
+            filename: 'combined.log',
```
Comment (shjortConcordium) [inline src/logger.ts:11]:
```suggestion
            filename: minimist(process.argv.slice(2)).logLocation || 'combined.log',
```
If you want simple, but not that hardcoded?

File: src/index.ts [hunk 2]
```diff
@@ -26,9 +28,38 @@ const server = new jayson.Server(getJsonRpcMethods(nodeClient));
...
+            res.status(400);
+            res.send(error);
+            return;
+        }
+
+        if (response) {
+            logger.info('Successful request', { correlationId });
+            return res.send(response);
+        } else {
+            res.status(204);
```
[the remark this answers is not in the document]
Comment (shjortConcordium) [reply src/index.ts:54]: I GUESS NOT

[the remark this answers is not in the document]
Comment (orhoj) [reply src/index.ts:54]: I have added some logging for this case :-)

File: src/logger.ts (hunk 1 above)
Comment (orhoj) [reply src/logger.ts:11]: Good idea.

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**